### PR TITLE
Added all 1.15 bocks: bee_hive, beenest, honeycom_block and honey_bock

### PR DESCRIPTION
**Pull request by Gyzie**
Supports beehive/nests in all directions with or without a honey_state of 5.

image

Addresses #1694 

Mojang also decided to change the chest textures for some reason. I'll look into how we can support pre 1.15 textures and 1.15+, any suggestions? _Alright so this build breaks because of a missing 1.15 jar together with_ #1687

### CODE REVIEW

**Comment by Gyzie** [comment]
Updated travis to Minecraft version 1.15 to resolve building issues. When #1702 is merged this build should pass.

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,8 @@ docs/_build/
 Minecraft_Overviewer.egg-info
 terrain.png
 cachedir*
+debug_config.py
```
Comment (CounterPillow) [inline .gitignore]: please don't, you can keep those in your `.git/info/exclude` as they are specific to you

Comment (Gyzie) [reply .gitignore]: Reverted 👍

**Comment by CounterPillow** [comment]
Thanks, I cleaned up the git history and merged it as https://github.com/overviewer/Minecraft-Overviewer/commit/885fa75f54dc5059b5085f530af259194edd62ae

(Sadly, due to the small history rewrite, github will show this as "closed" instead of "merged", but rest assured it's in with you as the author)

**Pull request closed without merging**